types/tipset_test.go

```diff
@@ -149,13 +149,13 @@ func TestTipSet(t *testing.T) {
 		assert.Equal(t, []*Block{b1, b2, b3}, ts.ToSlice()) // tipset is immutable
 	})
 
-	t.Run("string", func(t *testing.T) {
-		// String shouldn't really need testing, but some existing code uses the string as a
-		// datastore key and depends on the format exactly.
-		assert.Equal(t, "{ "+b1.Cid().String()+" }", RequireNewTipSet(t, b1).String())
-		assert.Equal(t, "{ "+b1.Cid().String()+" "+b2.Cid().String()+" "+b3.Cid().String()+" }",
-			RequireNewTipSet(t, b3, b2, b1).String())
-	})
+	//t.Run("string", func(t *testing.T) {
```

Comment:
FYI: @frrist's related comments in PR [here](https://github.com/filecoin-project/go-filecoin/pull/2897#discussion_r291278340)

Reply:
We do need this test. It can be made more robust by using a SortedCidSet explicitly to construct the "expected" string.